### DISCUSSION TITLE
Error 404 when download Exports Filament

### DISCUSSION THREAD

**Discussion by jesslynowen7** [filamentphp/filament#14068] [Help]
### Package

Actions

### Package Version

v.3.2.x

### How can we help you?

I have an app that running properly in local and also server (using Filament 3.2), but i have an update to export a column with relationship which mean i need to update my FIlament to newest version because there is a bug for relationship column in early version of 3.2. All good in local. After that, i push to server but unfortunately the exports don't work. when i download the csv/xlsx file from notification, it keep give error 404. Can someone tell me whats happen and how to fix that. This is my repository jesslynowen7/calculator2
![error](https://github.com/user-attachments/assets/79262974-5ec4-4625-b8a1-4fd924638abb)

this is my htaccess on server
![ht](https://github.com/user-attachments/assets/5670c364-6c21-472a-a3f1-2256bf533ed8)

and this is my route/web.php on server
![route](https://github.com/user-attachments/assets/3c2a577e-a1f2-4a1f-b832-feb4c0245739)

**Comment by alisalehi1380** [comment]
any update?
